src/bundle/MarkupEditorView.scss

```diff
@@ -3,8 +3,8 @@
 $selection-color: rgba(98, 146, 255, 0.2);
 $toolbar-height: 28px;
 
-.ye-markup-editor,
-.ye-markup-preview {
+.g-md-markup-editor,
```

Comment:
question (not blocks):

What is the reasoning behind choosing a class name and what does -g stand for? Is this consistent with the Gravity UI?

Reply:
yes, `g-md-` its short for Gravity Markdown Editor